include/rgl/api/extensions/pcl.h

```diff
@@ -16,6 +16,17 @@
 
 #include <rgl/api/core.h>
 
+/******************************** GRAPH ********************************/
+
+/**
+ * Obtains the point cloud of any node in the graph and saves to given file.
+ * Output file will be saved in PCD format (https://pointclouds.org/documentation/tutorials/pcd_file_format.html)
+ * @param node Node to get point cloud from.
+ * @param file_path Path to the output pcd file.
```

Comment:
There are any requrements? Absolute? Relative?
That should be in param descriprion.

Reply:
There is no restriction. It could be absolute or relative.